alu_u32/src/lt/columns.rs

```diff
@@ -0,0 +1,29 @@
+use core::borrow::{Borrow, BorrowMut};
+use core::mem::{size_of, transmute};
+use valida_derive::AlignedBorrow;
+use valida_machine::Word;
+use valida_util::indices_arr;
+
+#[derive(AlignedBorrow, Default)]
+pub struct Lt32Cols<T> {
+    pub input_1: Word<T>,
+    pub input_2: Word<T>,
+
+    /// Boolean flags indicating which byte pair differs
+    pub byte_flag: [T; 3],
+
+    /// Bit decomposition of 128 + input_1 - input_2
+    pub bits: [T; 9],
+
+    pub output: T,
+
+    pub is_real: T,
```

Comment:
I guess since this is unconstrained, we could call it `multiplicity` to hint that non-boolean values are possible. Though unsure if it's worth the extra code and witness generation overhead, just an option for later.